libraries/botbuilder-dialogs/package.json

```diff
@@ -26,6 +26,7 @@
     "@microsoft/recognizers-text-suite": "1.1.4",
     "@types/node": "^10.12.18",
     "botbuilder-core": "4.1.6",
+    "botframework-expressions": "4.1.6",
```

Comment:
why did we need to add this?

Reply:
I see why you added this but we shouldn't be taking dependencies for a single interface.

---
In reply to: [363404165](https://github.com/microsoft/botbuilder-js/pull/1483#discussion_r363404165) [](ancestors = 363404165)